bundles/org.openhab.binding.openweathermap/README.md

```diff
@@ -135,7 +188,7 @@ The "3h" value will be divided by three to always have an estimated value for on
 | forecastToday, forecastTomorrow, forecastDay2, ... forecastDay16 | condition-id         | String               | Id of the forecasted weather condition. **Advanced**                       |
 | forecastToday, forecastTomorrow, forecastDay2, ... forecastDay16 | icon                 | Image                | Icon representing the forecasted weather condition.                        |
 | forecastToday, forecastTomorrow, forecastDay2, ... forecastDay16 | icon-id              | String               | Id of the icon representing the forecasted weather condition. **Advanced** |
-| forecastToday, forecastTomorrow, forecastDay2, ... forecastDay16 | apparent-temperature | Number:Temperature   | Forecasted apparent temperature.                                           |
+| forecastToday, forecastTomorrow, forecastDay2, ... forecastDay16 | apparent-temperature | Number:Temperature   | Forecasted apparent temperature. Not available in the One Call API         |
```

Comment:
Not sure if this is true. I have this channel available in my One Call API thing and the API should deliver a `feels_like` datapoint

Reply:
Dear @boehan , there are indeed feels_like points in the daily response, but those are the points for morning, day, eve and night. I have them included as apparent-morning, apparent-day, apparent-evening and apparent-night. But there is no "general" feels_like value in the data structure. However, I could use the apparent-day response value for this to be a little bit more compatible to the other API; what do you think?

Reply:
You are right. Seems I just mixed this up with the current weather data. From my point of view, I would leave it as is then. 
However, lokking at the "Daily Forecast 16 Days" API, it should be returning the same datapoints (and also no "general" `feels_like` temperature), so I'm not sure how this is handled in the original thing. If the `apparent-day` value is used here, this should be indicated in description.

Reply:
> ... I'm not sure how this is handled in the original thing...

seems to be using the "day" value here as well
https://github.com/openhab/openhab-addons/blob/6e3c6400434f0f490a1d4aaeafb4f1cb15d0368c/bundles/org.openhab.binding.openweathermap/src/main/java/org/openhab/binding/openweathermap/internal/handler/OpenWeatherMapWeatherAndForecastHandler.java#L461

So, IMO there are two options:
1. use the "day" value here and make this clear in the description, since it is the case for both forecast things.
2. add the individual datapoints (day, morning, evening, night) to the original forecast API thing as well

The most feature complete approach would be option 2. However, as removing the `apparent-temperature` channel from the original forecast would be a breaking change, I would suggest to leave it and adapt the description thereof anyways.

Reply:
Dear @boehan , I would prefer to go for option 1 for the moment, because my primary goal is to get the oncall API handler done. Option 2 can be done independent from that as it affects the "classic" openweathermap API. Maybe I do a separate small PR for that, but as you may have noticed from my delayed responses, I currently don't have much spare time to spend on this binding.

Reply:
Maybe it's better to just adapt the description and not duplicate the `apparent-day` value here. But I'm not a maintainer, so that's just my opinion.
```suggestion
| forecastToday, forecastTomorrow, forecastDay2, ... forecastDay16 | apparent-temperature | Number:Temperature   | Forecasted apparent day temperature. Not available in the One Call API (see apparent-day channel)        |
```

Reply:
Did it that way with a slightly different comment. Sorry, saw your suggestion only after I already committed it.